columnflow/selection/__init__.py

```diff
@@ -275,13 +275,17 @@ def __iadd__(self: SelectionResult, other: SelectionResult | None) -> SelectionR
 
         # helper to create a view without behavior
         def deepcopy_without_behavior(struct: T) -> T:
-            return copy.deepcopy(law.util.map_struct(
-                (lambda obj: ak.Array(obj, behavior={}) if isinstance(obj, ak.Array) else obj),
+            return law.util.map_struct(
+                (lambda obj: (
+                    ak.Array(obj, behavior={})
+                    if isinstance(obj, ak.Array)
+                    else copy.deepcopy(obj)
+                )),
```

Comment:
The `deepcopy` was in the wrong spot before. Also, ak arrays do no longer support deepcopying.